.github/workflows/performance.yaml

```diff
@@ -0,0 +1,34 @@
+name: "Performance Tests"
+
+on:
+  pull_request:
+    branches: [ main, develop ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: [self-hosted, wind-tunnel]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install nix
+        uses: cachix/install-nix-action@v27
+
+      - uses: cachix/cachix-action@v15
+        with:
+          name: holochain-wind-tunnel
+          extraPullNames: holochain-ci
+          authToken: ${{ secrets.CACHIX_HOLOCHAIN_WIND_TUNNEL }}
+
+      - uses: Swatinem/rust-cache@v2
+
+      - name: Smoke test - zome_call_single_value
+        run: |
+          # Start a sandbox conductor and run it in the background
+          nix develop .#ci -c bash -c "hc s clean && echo "1234" | hc s --piped create && echo "1234" | hc s --piped -f 8888 run &"
+
+          # TODO using `localhost` is resolving to an IPv6 address, but why is that giving a connection refused?
```

Comment:
Holochain is supposed to listen on both IPv4 and IPv6 because we had this issue when GitHub runners switched over to using IPv6 by default.

Reply:
This was actually a copy-paste of a comment in the test workflow :sweat_smile: I just removed the comment as it doesn't seem to be an issue here, maybe it can also be removed from the GitHub-hosted runner tests now too?

https://github.com/holochain/wind-tunnel/blob/5f376c0c3cdaaa554818b021f81fd58f9f57804b/.github/workflows/test.yaml#L75

Reply:
Yep, definitely!